doc/changes/dev/13428.bugfix.rst

```diff
@@ -0,0 +1 @@
+Allow `first_samp` and true number of events (before dropping epochs) to be considered when exporting to EEGLAB format. by `Scott Huberty`_
```

Comment:
Unless you actually wanted it to link to a `first_samp` entry somewhere, I think you wanted code mode, i.e., double-backticks (different for reST than markdown)
```suggestion
Allow ``first_samp`` and true number of events (before dropping epochs) to be considered when exporting to EEGLAB format. by `Scott Huberty`_
```